src/main/kotlin/com/danix/anaconda/QuoteController.kt

```diff
@@ -14,6 +14,7 @@ import javax.servlet.http.HttpServletRequest
 import javax.ws.rs.core.Response.Status
 
 @RestController
+@RequestMapping("/api")
```

Comment:
The reason I added this, is that I only want to have security for the `/api`, and not for the other endpoints. The plan #14 would be to add a `/health` endpoint for example, without auth. Let's see how the project evolves, and add it later as soon as we have more controllers.